JitsiConference.js

```diff
@@ -533,6 +543,25 @@ JitsiConference.prototype.replaceTrack = function (oldTrack, newTrack) {
 };
 
 /**
+ * Replaces the tracks at the lower level by going through the Jingle session
+ * and WebRTC peer connection. The method will resolve immediately if there is
+ * currently no JingleSession started.
+ * @param {JitsiLocalTrack|null} oldTrack the track to be removed during
+ * the process or <tt>null</t> if the method should act as "add track"
+ * @param {JitsiLocalTrackn|null} newTrack the new track to be added or
+ * <tt>null</tt> if the method should act as "remove track"
+ * @return {Promise}
+ * @private
+ */
+JitsiConference.prototype._doReplaceTrack = function (oldTrack, newTrack) {
+    if (this.jingleSession) {
+        return this.jingleSession.replaceStream(oldTrack, newTrack);
```

Comment:
Not really, because the underlying pipe starting at JingleSessionPC accepts either JitsiLocalTrack or MediaStream. I have plans on fixing that in next PRs, but as long as it accepts either one it will always be confusing. Unless I call it reaplceStreamOrTrack